apps/system/utils/Kconfig

```diff
@@ -81,6 +81,14 @@ config ENABLE_HEAPINFO
 	---help---
 		Show information about memory status per thread
 
+config ENABLE_IRQINFO
+	bool "irqinfo"
+	default y
```

Comment:
This spends memory to save name, count and so on. In my opinion, I don't want to enable this defaultly.